docs/mkdocs.yml

```diff
@@ -330,11 +330,10 @@ nav:
       - Environment variables: cloud/reference/env_var.md
 
   - Examples:
-    - tutorials/langgraph-platform/local-server.md
```

Comment:
For some reason, including the page here causes the issue. I suspect `mkdocs` is just taking the location of the last reference to the page (`Examples`) and using the navigation from there.

In other words, you can't reference a page twice in `mkdocs.yml`?

Reply:
Confirmed. It's not recommended to reference a page twice in `mkdocs.yml`.